model_compression_toolkit/target_platform_capabilities/target_platform/op_quantization_config.py

```diff
@@ -177,6 +258,61 @@ def clone_and_edit(self, **kwargs):
             self.__edit_quantization_configuration(qc, kwargs)
         return qc_options
 
+    def clone_and_edit_weight_attribute(self, attrs: List[str] = None, **kwargs):
```

Comment:
For kwargs or return type?
basically, it is also missing in all other clone_and_edit methods in these classes, and the return type can't be specified because it is actually a constructor.